### DISCUSSION TITLE
Thumbnails

### DISCUSSION THREAD

**Discussion by shipgimp** [sumatrapdfreader/sumatrapdf#2679] [General]
I'm really missing the thumbnails for my pdfs. 
They were fine a week or so ago then after an update they're just the yellow and black icon.
Am I missing something?
Thanks

**Comment by GitHubRulesOK** [comment]
thumbnails are an install option and can hiccup when windows prevhost is occupied on something else
- Best to back up settings (they should not be deleted but best to be safe)
- clean uinstall and re-install with the preview option selected
- 
![image](https://user-images.githubusercontent.com/12754913/172182181-571bf326-a384-4eed-bc64-274fde9b2f29.png)

**Reply by Nozokyo** [reply]
I use the Portable version, which starts instantly, with no install options.

I this stored somewhere in the **settings.txt** file?
Or do I need to use the install version?

**Reply by GitHubRulesOK** [reply]
@Nozokyo 
Thumbnails and text searching are WINDOWS functions thus any dlls Microsoft, Adobe or SumatraPDF need to be installed for windows to USE them.

**Reply by rudolphos** [reply]
I have no other PDF software installed. But I have reinstalled SumatraPDF many times with both options checked, unchecked, etc. No thumbnails are showing. The only options is to use powertoys pdf thumbnail generator (Powertoys.pdfthumnailprovider) - which is slow to the point it actually shows spinning circle cursor when pdf thumbnails are generating. This happens on Win11 25h2.

**Reply by kjk** [reply]
which version of SumatraPDF?
where do you expect them to show up? explorer?

**Reply by rudolphos** [reply]
@kjk 
Sumatra 3.6.17242 pre-release 64bit. I expect them to show up in the regular file explorer and Everything search tool. I've tried many things:
- disabled powertoys pdf thumbnailer; tried with powertoys uninstalled entirely;
- reinstalled for all users, for one user; 
- reinstalled with both lower options checked and with one lower checked;
- cleaned the entire thumbnail cache after every action;
- used BulkCrapUninstaller to remove SumatraPDF and leftovers with rebooting;
- tried `regsvr32 "C:\Program Files\SumatraPDF\PdfPreview.dll"` with some additional other cmds
- unblocked several PDFs from the web to see if that makes a difference;
- added sumatrapdf.exe and all dll files into antivirus/defender exclusions list + added .pdf files in exclusion list;
- `HKEY_CURRENT_USER\Software\Policies\Microsoft\Windows\Explorer` - added DWORD (32-bit) Value `AllowCloudThumbnails`: 1 and `DisableThumbnailsOnNetworkFolders`: 0; restarted
- from shellexview, opened regedit CLSID settings:
<img width="1089" height="264" alt="image" src="https://github.com/user-attachments/assets/bbbe9cd4-3e3a-4a2e-bb56-8f2088ba81a4" />
<img width="890" height="253" alt="image" src="https://github.com/user-attachments/assets/8f0ff425-d3dc-4186-bc6f-7a7ca05260a6" />

...but nothing has worked so far. Powertoys thumbnailer for some reason is the only one that is 100% reliable and works as soon as it's toggled on.

**Reply by Evanshion** [reply]
Hi, same problem here, installed a standard version of Sumatra (3.6.1 64bit, win 10) and I can't see thumbnails.
is there some fix or option? 
I checked the option during the installation.
Sumatra is the only pdf reader installed.
It is setted as default for pdf.

**Reply by GitHubRulesOK** [reply]
@Evanshion I am not the developer so can't say why but In my uses I install for all users as admin and that works most of the time.
But if I install just for myself the preview works but thumbnails don't and that may also apply to pre-release on Windows 10, unknown if MS have been messing about with user access rights again but they changed PDF preview some time ago as part of heightened security!

**Reply by Evanshion** [reply]
I solved. The files were on a cloud so I can see only the shadow images...
my bad :p
I copied them on my local and all works!
Thanks!

**Comment by pitter7488258450-cmyk** [comment]
I’ve been tweaking my Sumatra settings for this same reason—the default white is definitely too bright for long reading sessions.
Changing the BackgroundColor in the Advanced Options (the 'FixedPageUI' section) usually does the trick for PDFs. If you're looking for good hex codes for subtle, eye-friendly shades that aren't just pure black or white, I've used [Solid Color Wallpapers](https://solidbg.com/) to find a few solid 'paper' and 'sepia' tones that look great behind text.
Definitely try a soft grey or off-white if you want to keep it readable but save your eyes some strain!

**Reply by GitHubRulesOK** [reply]
@pitter7488258450-cmyk 
This thread is about preview thumbnails  not theming but theming is easy in pre-release you dont need to fiddle unless adjusting the many built in ones 
<img width="494" height="272" alt="image" src="https://github.com/user-attachments/assets/3a4766e5-0555-42d5-a923-5735d545597e" />

**Comment by filchef** [comment]
Future request - for different file extensions - different colors for  Thumbnails

**Comment by GitHubRulesOK** [comment]
@filchef it is not always that simple since when you select a file type to be assigned to a DEFAULT app it is Windows uses the app icon (ONLY one style) it can be different for smallest icons without any change to the system just the icon size as details see below where Windows uses in my case the green registered epub icon.
<img width="326" height="235" alt="image" src="https://github.com/user-attachments/assets/c5a8d0e5-603d-4ce5-84a1-61ccc677929d" />

<img width="547" height="177" alt="image" src="https://github.com/user-attachments/assets/86c95ca2-c562-4e4b-85c2-1aea5c952768" />

<img width="408" height="122" alt="image" src="https://github.com/user-attachments/assets/919b5daf-4657-46fc-b956-a80631fdb9f3" />
